config/initializers/log_version.rb

```diff
@@ -0,0 +1,2 @@
+# Output version to log on startup
+Rails.logger.info "Starting RMT server version #{RMT::VERSION}"
```

Comment:
Should I be seeing this message when running `rails s`? Because I don't see it.
The file IS getting loaded though. Putting a `puts` there, I see it.

Reply:
You are not seeing it, because in development mode logs are written to a file.
Run in production mode and you'll see it on `STDOUT`.

Reply:
Huh, okay. I found it strange because, at least locally, I also see all logging in the console.

Reply:
`Rails.logger` writes to a file in dev mode, but other logger instances (`rmt-cli` an such) might always log to `STDOUT`.
Maybe things like `rmt-cli mirror` and `sync` should report their versions as well?